content/en/blog/2022/gc-elections-2022.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Announcing the 2022 OpenTelemetry Governance Committee Election
+linkTitle: OpenTelemetry Governance Committee Election 2022
+date: 2022-09-14
+author: OpenTelemetry Governance Committee
+---
+
+The OpenTelemetry project is excited to announce the 2022 OpenTelemetry
+Governance Committee (GC) election. Nominations are due by end-of-day on 7
+October 2022, with the ratification happening by 8 October 2022 and a list of
+eligible candidates will be shared on 9 October 2022. Voting will take place
+between 18 October 2022 and 20 October 2022, and the final election results will
+be announced 22 October 2022.
+
+## Vote!
```

Comment:
Heh, on mobile when I read this, I thought it said `Votel` and I was going to applaud the pun.